content/en/docs/v3.6/dev-guide/api_reference_v3.md

```diff
@@ -478,15 +538,28 @@ Empty field.
 
 | Field | Description | Type |
 | ----- | ----------- | ---- |
+| (versionpb.etcd_version_msg) |  | option |
 | header |  | ResponseHeader |
 | version | version is the current cluster version. | string |
 
 
 
+##### message `DowngradeVersionTestRequest` (api/etcdserverpb/rpc.proto)
+
+DowngradeVersionTestRequest is used for test only. The version in this request will be read as the WAL record version.If the downgrade target version is less than this version, then the downgrade(online) or migration(offline) isn't safe, so shouldn't be allowed.
+
+| Field | Description | Type |
+| ----- | ----------- | ---- |
+| (versionpb.etcd_version_msg) |  | option |
+| ver |  | string |
+
+
+
```

Comment:
Could you please remove these two empty lines? This is triggering the linter violation.